src/services/powerbi/PowerBIService.js

```diff
@@ -5,13 +5,19 @@ import { GET_EMBED_INFO_URL } from '../../state/commons/PowerBIConstants';
 import { clientApi } from '../ClientApi';
 import { COSMOTECH_API_SCOPE, POWER_BI_API_DEFAULT_SCOPE } from '../config/Auth';
 import { EmbedConfig, PowerBiReportDetails } from './PowerBIModels';
-import { handleServiceAccountError, handleUserAccountError, PowerBIError } from './errors';
+import { handleServiceAccountError, handleUserAccountError, PowerBIError, forgePowerBIError } from './errors';
 
 /**
  * Get PowerBI accessible data
  * @return Details like Embed URL, Access token and Expiry, errors if any
  */
 const getPowerBIData = (powerBIWorkspaceId, reportsIds, logInWithUserCredentials) => {
+  if (process.env.REACT_APP_API_KEY)
+    return {
+      accesses: null,
+      error: forgePowerBIError('', 'PowerBI disabled', 'API key detected, PowerBI dashboards have been disabled'),
+    };
+
```

Comment:
If we use the API key, is it a real error ? :thinking: I think it can be confusing for end users to see a red error banner while this connection method can't be configured to use PowerBi. Maybe, a `console.error` for integrators is less intrusive? The chances to get this use case are low because an integrator won't configure powerBi with API key so disabled option will come first but it can happen if the new workspace is a copy of an existing one with powerBi :slightly_smiling_face:

Reply:
End-users will probably never show this error message, because we must only use API keys with local webapps (otherwise, the API key could leak _via_ the headers of the API queries).
Since only devs will see this error message, I thought a banner would be more straightforward to show why the charts are disabled, even if the cause comes from the configuration and it's not really an error :sweat_smile:

Reply:
In this case, we can keep an error banner :slightly_smiling_face: